crypto/cipher/encrypter/encrypter.go

```diff
@@ -0,0 +1,30 @@
+package encrypter
+
+import (
+	crypto "github.com/mailchain/mailchain/crypto/cipher"
+	"github.com/mailchain/mailchain/crypto/cipher/aes256cbc"
+	"github.com/mailchain/mailchain/crypto/cipher/nacl"
+	"github.com/pkg/errors"
+)
+
+// Cipher Name lookup
+const (
+	NoOperation string = "noop"
+	NACL        string = "nacl"
+	AES256CBC   string = "aes256cbc"
+	EmptyCipher string = "empty"
+)
+
+// GetEncrypter is an `Encrypter` factory that returns an encrypter
+func GetEncrypter(encryption string) (crypto.Encrypter, error) {
+	switch encryption {
+	case AES256CBC:
+		return aes256cbc.NewEncrypter(), nil
+	case NACL:
+		return nacl.NewEncrypter(), nil
+	case EmptyCipher:
```

Comment:
`case "":`  is more likely if someone is not supplying an argument. Although an empty string is caught in the hander I would expect this method to be called in other places in the future.

Reply:
Alright cool, made the changes!